src/CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 file(GLOB_RECURSE srcs *.cpp)
 add_executable(vera ${srcs})
+target_compile_options(vera PRIVATE -O3)
```

Comment:
-O3 is notorious to enable unstable optimization, using `-O2` would be safer and shouldn't make a huge difference in final speed.

Reply:
O3 is totaly safe nowadays ... We are not in 1999 anymore

Reply:
No, O3 does not enable unsafe optimizations, while Ofast does.
O3 is as safe as other optimizations flags since GCC 2.8.
A good exemple of such safety is that it's used in some parts of the linux kernel, which is supported by old compiler, without impacting stability.
However the difference between O2 and O3 is still pretty significant and should not be ignored, even more for a non-critical program like this one.